base/task.jl

```diff
@@ -166,11 +166,21 @@ end
     elseif field === :exception
         # TODO: this field name should be deprecated in 2.0
         return t._isexception ? t.result : nothing
+    elseif field === :sticky
+        return getfield(t, :sticky_count) != 0
     else
         return getfield(t, field)
     end
 end
 
+function setproperty!(t::Task, field::Symbol, x)
+    if field === :sticky
+        t.sticky_count = convert(Bool, x)
```

Comment:
This is rather a compat layer for supporting `.sticky` property.

`@async` task is never marked as unsticky because (1) its `sticky_count` is initialized to 1 (in the C function `jl_new_task`) and (2) a decrement (if any) is always paired with a preceding increment.